### PR TITLE
[ADD] website_event_sale_b2x_alt_price: alt B2C/B2B price on website events

### PR DESCRIPTION
**Pull request by yajo**
This module extends the functionality of website event sales to support
displaying the alternative price in the website event registration page and to
allow your visitors to know the final price in case they will have some special
fiscal tax behavior.

screenshot-2021-04-28-13-26-45

@Tecnativa TT29393

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Please add e-commerce to oca_depedencies.txt

**Comment by yajo** [comment]
Thanks, done.

**Comment by pedrobaeza** [comment]
Travis is red

**Comment by yajo** [comment]
Linters... I'm too used to pre-commit these days sorry 🙈

**Comment by pedrobaeza** [comment]
I'm not seeing both prices in the register page:

Selección_002

**Comment by yajo** [comment]
Did you make sure the product attached to the event tickets has some kind of taxes?

**Review by pedrobaeza — APPROVED** [review]
Seen that is working. @chienandalu or @sergio-teruel can you finish your review?

File: website_event_sale_b2x_alt_price/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from . import models
+# from . import wizards
+# from . import controllers
+# from . import report
+# from .hooks import pre_init_hook, post_load, post_init_hook, uninstall_hook
```
Comment (sergio-teruel) [inline website_event_sale_b2x_alt_price/__init__.py]: Remove commented lines

**Review by sergio-teruel — APPROVED** [review]
Little comment only

**Comment by yajo** [comment]
Thanks, all attended.

/ocabot merge nobump

**Pull request merged**